src/Utility/Updater.lua

```diff
@@ -0,0 +1,40 @@
+--!nonstrict
+
+--[[
+    Centralizes all RunService connections so that the various parts of Fusion 
+    don't connect to RunService independently of one another.
+
+    You can optionally pause all internal updating by calling `.pause()`, and
+    then resume updating by calling `.resume()`.
+]]
+
+local RunService = game:GetService("RunService")
+
+local Updater = {}
```

Comment:
Properties can be initialised in the table literal. This is more clear than doing it on the lines below, and again more in line with the rest of the Fusion codebase.